logout.md

```diff
@@ -0,0 +1,17 @@
+# Logout
+
+The integration must implement the `/logout` endpoint.  The endpoint is used to
+delete the [Authentication Cookies](cookie-authentication.md) that were set on
+login.
+
```

Comment:
We should also make a note to remove access tokens and refresh tokens at stormpath for non Id site logout